gst/nnstreamer/hw_accel.c

```diff
@@ -24,7 +24,16 @@
 #endif /* __TIZEN__ */
 #endif /* __arch64__ || __arm__ */
 
+#if !defined(__APPLE__)
```

Comment:
Oh, we can use ```__APPLE__``` to distinguish macOS from the platforms. I need to change ```-D__MACOS__``` provided by the meson script.